enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -236,6 +233,27 @@ public void activateUser( @Name( "username" ) String username ) throws IOExcepti
 
         subject.getUserManager().assertAndGetUser( username );
 
+        return terminateSessionsForValidUser( username );
+    }
+
+    // ----------------- helpers ---------------------
+
+    private Stream<TransactionResult> terminateTransactionsForValidUser( String username )
+    {
+        Long killCount = 0L;
+        for ( KernelTransaction tx : getActiveTransactions() )
+        {
+            if ( tx.mode().name().equals( username ) )
```

Comment:
This will miss transactions that are currently executing procedure calls, since they will have a restricted access mode